lib/dradis/plugins/templates.rb

```diff
@@ -3,13 +3,31 @@ module Plugins
     module Templates
       extend ActiveSupport::Concern
 
+      LEGACY_FIELDS_REGEX = /%(\S+?)%/.freeze
```

Comment:
It's worth adding a timestamp here as a comment, i.e. introduced in 2024 migration, good to delete in 2026 or whatever.